whisper_cpp_vendor/CMakeLists.txt

```diff
@@ -34,20 +34,38 @@ set_target_properties(
 
 # install ggml header
 install(
-  FILES ${whisper_SOURCE_DIR}/ggml.h
+  FILES ${whisper_SOURCE_DIR}/ggml/include/ggml.h
   DESTINATION include
 )
 ##############
 # end of fixes
 ##############
+# Export header files for downstream packages
+install(
+  DIRECTORY
+    ${whisper_SOURCE_DIR}/include/
+    ${whisper_SOURCE_DIR}/ggml/include/
+  DESTINATION include
+)
+
+install(
+  TARGETS ggml
+  EXPORT export_whisper
+  LIBRARY DESTINATION lib
+  INCLUDES DESTINATION include
+)
 
 ament_export_targets(export_whisper HAS_LIBRARY_TARGET)
 
+# Install the library
 install(
   TARGETS whisper
   EXPORT export_whisper
   LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  RUNTIME DESTINATION bin
```

Comment:
does whisper.cpp compile binaries?

Reply:
I think no, I have removed the two lines and it compiles and runs.  I committed it as a change to the branch.

Reply:
As a separate question, do you possibly have insight on a warning that appears when compiling:
```
--- stderr: whisper_cpp_vendor                                                                 
-- CUDA host compiler is GNU 11.4.0
INSTALL TARGETS - target ggml has PUBLIC_HEADER files but no PUBLIC_HEADER DESTINATION.
---
```

Reply:
no sorry, don't know. Maybe https://stackoverflow.com/a/59257505 ?

Reply:
Hmm, ok I will close this for now and look into it at a later point since it isn't breaking and can raise it as an issue.

Thanks for the link, at a short glance I saw some useful information for my understanding of CMake.